examples/experimental/persistent-volume-provisioning/README.md

```diff
@@ -79,7 +94,7 @@ will require that this value matches what is configured by the administrator.
   "metadata": {
     "name": "claim1",
     "annotations": {
-        "volume.alpha.kubernetes.io/storage-class": "foo"
```

Comment:
what is happening to the old annotation?  Are we abandoning it?  Can we allow the old annotation with the old behavior for 1 overlapping release and then remove it?  We'll need a transition doc.